tsconfig.json

```diff
@@ -1,17 +1,16 @@
 {
-  "compileOnSave": false,
-  "compilerOptions": {
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "target": "ES5",
-    "sourceMap": true,
-    "jsx": "react",
-    "jsxFactory": "Snabbdom.createElement",
-    "noImplicitReturns": true,
-    "strict": true,
-    "outDir": "lib",
-    "strictFunctionTypes": false,
-    "lib": ["ES6", "ES5", "DOM"]
-  },
-  "include": ["src/index.ts"]
+    "compileOnSave": false,
```

Comment:
Same here as with package.json, please don't reformat as it makes the actual changes hard to see